README.md

```diff
@@ -46,6 +46,40 @@ SKIP_AUTH=1 FORMS_ADMIN_URL='http://localhost:3000/' PRODUCT_PAGES_URL='http://l
 The end to end tests can be run without visiting the product pages by setting
 the `SKIP_PRODUCT_PAGES` environment variable to `1`.
 
+### Skipping the s3 submission test
+
+The end to end tests can be run without testing a form with the `submission_type` of `s3` by setting the `SKIP_S3` environment variable to `1`.
+
+### Running the s3 submission test
+
+- in `forms-runner`: 
+    - add your govuk_notify.api_key and iam_role_arn (this will be the ARN of the role with permissions to upload to the s3 bucket in the deploy environment) to the settings.yml
+    - start the server using an iam role that can assume the above role (eg: `gds aws forms-dev-readonly -- bundle exec rails s`)
+- in `forms-admin` 
+    - add your govuk_notify.api_key to the settings.yml 
+    - start the server (without aws)
+- in `forms-api`: 
+    - ensure the seeded s3 submission test form is set up correctly, and run the following rake task:
+        - `rake "forms:set_submission_type_to_s3[2, ${the name of the submission bucket}, ${the aws account id where the bucket lives}, ${the region}]"`
+    - start the server (without aws)
+- in `forms-e2e-tests`
+    - start an aws shell:
+        - `gds aws forms-dev-readonly --shell`
+    - run the end to end tests tests:
+``` 
+SKIP_AUTH=1 \
+FORMS_ADMIN_URL='http://localhost:3000/' \
+PRODUCT_PAGES_URL='http://localhost:3002/' \
+SKIP_PRODUCT_PAGES=1 \
+LOG_LEVEL=info \
+SETTINGS__GOVUK_NOTIFY__API_KEY= ${ your notify api key here } \
+FORMS_RUNNER_URL='http://localhost:3001/' \
+AWS_S3_ROLE_ARN= ${ the iam role arn } \
```

Comment:
It might be worth giving this environment variable the same name as what it would be for forms-runner:

```suggestion
SETTINGS__AWS_S3_SUBMISSIONS__IAM_ROLE_ARN= ${ the iam role arn } \
```

Reply:
For this role in particular it would be good to specify somewhere that this has to the the role created for the e2e tests, as that's the only one that has the permission to delete from the bucket. I was trying to test this and it seems I can't assume that role from my account.

Do we either need to make it possible to assume the e2e role, or add the permission to delete to the other role?

Either way, as @lfdebrux said, it would probably be good to link out to somewhere we can detail the role, bucket, etc so people don't have to go searching around. It might be fine to put this in the wiki?

Reply:
good idea - done

Reply:
@stephencdaly btw I've deployed that change so you should be able to assume the role now